modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js

```diff
@@ -246,9 +247,9 @@ class Sidebar extends Component {
 				<div class={styles} ref="container">
 					<div class="sidebar sidebar-light">
 						<nav class="component-tbar tbar">
-							<div class="container-fluid">
+							<ClayLayout.ContainerFluid>
```

Comment:
Hey @eduardoallegrini, I'm not sure about this but I see that the default of `ContainerFluid` is `container-fluid container-fluid-max-xl` can this cause any collateral effect for this case?

Reply:
Yeah I know it's not exactly the same but I think it's hardly probable that someone is going to need a space wider than 1280px, so I prefered to limit it thinking it was an error/missing-class in the first place. If somebody ask for this we can fix it easily

Reply:
Sure, no problem. Just a concern if it causes any side effects visually.